### PR TITLE
sync main branch to develop branch (#692)

### PR DESCRIPTION
**Pull request by sayaliM0412**
* Bump whelk-io/maven-settings-xml-action from 20 to 21

Bumps [whelk-io/maven-settings-xml-action](https://github.com/whelk-io/maven-settings-xml-action) from 20 to 21.
- [Release notes](https://github.com/whelk-io/maven-settings-xml-action/releases)
- [Commits](https://github.com/whelk-io/maven-settings-xml-action/compare/v20...v21)

---
updated-dependencies:
- dependency-name: whelk-io/maven-settings-xml-action dependency-type: direct:production update-type: version-update:semver-major ...

* Bump liquibase-github-actions/drop-all from 4.24.0 to 4.25.0

Bumps [liquibase-github-actions/drop-all](https://github.com/liquibase-github-actions/drop-all) from 4.24.0 to 4.25.0.
- [Release notes](https://github.com/liquibase-github-actions/drop-all/releases)
- [Commits](https://github.com/liquibase-github-actions/drop-all/compare/v4.24.0...v4.25.0)

---
updated-dependencies:
- dependency-name: liquibase-github-actions/drop-all dependency-type: direct:production update-type: version-update:semver-minor ...

* Bump actions/github-script from 7.0.0 to 7.0.1

Bumps [actions/github-script](https://github.com/actions/github-script) from 7.0.0 to 7.0.1.
- [Release notes](https://github.com/actions/github-script/releases)
- [Commits](https://github.com/actions/github-script/compare/v7.0.0...v7.0.1)

---
updated-dependencies:
- dependency-name: actions/github-script dependency-type: direct:production update-type: version-update:semver-patch ...

* Bump com.ibm.db2:jcc from 11.5.8.0 to 11.5.9.0

Bumps com.ibm.db2:jcc from 11.5.8.0 to 11.5.9.0.

---
updated-dependencies:
- dependency-name: com.ibm.db2:jcc dependency-type: direct:production update-type: version-update:semver-patch ...

* Bump org.codehaus.mojo:versions-maven-plugin from 2.16.1 to 2.16.2

Bumps [org.codehaus.mojo:versions-maven-plugin](https://github.com/mojohaus/versions) from 2.16.1 to 2.16.2.
- [Release notes](https://github.com/mojohaus/versions/releases)
- [Changelog](https://github.com/mojohaus/versions/blob/master/ReleaseNotes.md)
- [Commits](https://github.com/mojohaus/versions/compare/2.16.1...2.16.2)

---
updated-dependencies:
- dependency-name: org.codehaus.mojo:versions-maven-plugin dependency-type: direct:production update-type: version-update:semver-patch ...

* Bump org.postgresql:postgresql from 42.6.0 to 42.7.0

Bumps [org.postgresql:postgresql](https://github.com/pgjdbc/pgjdbc) from 42.6.0 to 42.7.0.
- [Release notes](https://github.com/pgjdbc/pgjdbc/releases)
- [Changelog](https://github.com/pgjdbc/pgjdbc/blob/master/CHANGELOG.md)
- [Commits](https://github.com/pgjdbc/pgjdbc/compare/REL42.6.0...REL42.7.0)

---
updated-dependencies:
- dependency-name: org.postgresql:postgresql dependency-type: direct:development update-type: version-update:semver-minor ...

* Bump org.firebirdsql.jdbc:jaybird from 5.0.2.java11 to 5.0.3.java11

Bumps [org.firebirdsql.jdbc:jaybird](https://github.com/FirebirdSQL/jaybird) from 5.0.2.java11 to 5.0.3.java11.
- [Release notes](https://github.com/FirebirdSQL/jaybird/releases)
- [Commits](https://github.com/FirebirdSQL/jaybird/commits)

---
updated-dependencies:
- dependency-name: org.firebirdsql.jdbc:jaybird dependency-type: direct:production update-type: version-update:semver-patch ...

* using Codex-/return-dispatch to get the run-id for OSS when a run is triggered (#666)

* fix error unexpected end of the stream within a double quoted scalar (#667)

* Bump org.xerial:sqlite-jdbc from 3.44.0.0 to 3.44.1.0

Bumps [org.xerial:sqlite-jdbc](https://github.com/xerial/sqlite-jdbc) from 3.44.0.0 to 3.44.1.0.
- [Release notes](https://github.com/xerial/sqlite-jdbc/releases)
- [Changelog](https://github.com/xerial/sqlite-jdbc/blob/master/CHANGELOG)
- [Commits](https://github.com/xerial/sqlite-jdbc/compare/3.44.0.0...3.44.1.0)

---
updated-dependencies:
- dependency-name: org.xerial:sqlite-jdbc dependency-type: direct:development update-type: version-update:semver-patch ...

* Bump commons-io:commons-io from 2.15.0 to 2.15.1

Bumps commons-io:commons-io from 2.15.0 to 2.15.1.

---
updated-dependencies:
- dependency-name: commons-io:commons-io dependency-type: direct:production update-type: version-update:semver-patch ...

* Bump org.mariadb.jdbc:mariadb-java-client from 3.3.0 to 3.3.1

Bumps [org.mariadb.jdbc:mariadb-java-client](https://github.com/mariadb-corporation/mariadb-connector-j) from 3.3.0 to 3.3.1.
- [Release notes](https://github.com/mariadb-corporation/mariadb-connector-j/releases)
- [Changelog](https://github.com/mariadb-corporation/mariadb-connector-j/blob/master/CHANGELOG.md)
- [Commits](https://github.com/mariadb-corporation/mariadb-connector-j/compare/3.3.0...3.3.1)

---
updated-dependencies:
- dependency-name: org.mariadb.jdbc:mariadb-java-client dependency-type: direct:development update-type: version-update:semver-patch ...

* Bump actions/setup-java from 3 to 4

Bumps [actions/setup-java](https://github.com/actions/setup-java) from 3 to 4.
- [Release notes](https://github.com/actions/setup-java/releases)
- [Commits](https://github.com/actions/setup-java/compare/v3...v4)

---
updated-dependencies:
- dependency-name: actions/setup-java dependency-type: direct:production update-type: version-update:semver-major ...

* Bump org.apache.maven.plugins:maven-javadoc-plugin from 3.6.2 to 3.6.3

Bumps [org.apache.maven.plugins:maven-javadoc-plugin](https://github.com/apache/maven-javadoc-plugin) from 3.6.2 to 3.6.3.
- [Release notes](https://github.com/apache/maven-javadoc-plugin/releases)
- [Commits](https://github.com/apache/maven-javadoc-plugin/compare/maven-javadoc-plugin-3.6.2...maven-javadoc-plugin-3.6.3)

---
updated-dependencies:
- dependency-name: org.apache.maven.plugins:maven-javadoc-plugin dependency-type: direct:production update-type: version-update:semver-patch ...

* Bump org.postgresql:postgresql from 42.7.0 to 42.7.1

Bumps [org.postgresql:postgresql](https://github.com/pgjdbc/pgjdbc) from 42.7.0 to 42.7.1.
- [Release notes](https://github.com/pgjdbc/pgjdbc/releases)
- [Changelog](https://github.com/pgjdbc/pgjdbc/blob/master/CHANGELOG.md)
- [Commits](https://github.com/pgjdbc/pgjdbc/compare/REL42.7.0...REL42.7.1)

---
updated-dependencies:
- dependency-name: org.postgresql:postgresql dependency-type: direct:development update-type: version-update:semver-patch ...

* Bump org.liquibase.ext:liquibase-sdk-maven-plugin

Bumps [org.liquibase.ext:liquibase-sdk-maven-plugin](https://github.com/liquibase/liquibase-sdk-maven-plugin) from 0.10.22 to 0.10.23.
- [Release notes](https://github.com/liquibase/liquibase-sdk-maven-plugin/releases)
- [Commits](https://github.com/liquibase/liquibase-sdk-maven-plugin/compare/v0.10.22...v0.10.23)

---
updated-dependencies:
- dependency-name: org.liquibase.ext:liquibase-sdk-maven-plugin dependency-type: direct:production update-type: version-update:semver-patch ...

* Bump net.snowflake:snowflake-jdbc from 3.14.3 to 3.14.4

Bumps [net.snowflake:snowflake-jdbc](https://github.com/snowflakedb/snowflake-jdbc) from 3.14.3 to 3.14.4.
- [Release notes](https://github.com/snowflakedb/snowflake-jdbc/releases)
- [Changelog](https://github.com/snowflakedb/snowflake-jdbc/blob/master/CHANGELOG.rst)
- [Commits](https://github.com/snowflakedb/snowflake-jdbc/compare/v3.14.3...v3.14.4)

---
updated-dependencies:
- dependency-name: net.snowflake:snowflake-jdbc dependency-type: direct:development update-type: version-update:semver-patch ...

* DAT-16380 (#673)

* using -Dliquibase.sdk.workflowId=run-tests.yml

* use sdk-maven-plugin 0.10.23

* get liquibase --version

* get branch develop

* Trigger build

---------

---------

### CODE REVIEW

**Review by KushnirykOleh — CHANGES_REQUESTED** [review]
we cannot merge this PR with Daniel's changes for TinyInt until liquibase-core is released. 
Builds on the main branch in test-harness use the released version of liquibase-core

**Review by KushnirykOleh — COMMENTED** [review]
good to merge after resolving the conflict

**Comment by sayaliM0412** [comment]
> good to merge after resolving the conflict

Conflicts resolved. Please verify

**Pull request merged**
